File: Web Programming/Assignment6/server/model/product.php
<?php
class Product {
    private $name;
    private $category;
    private $price;

    public function __construct(string $name, string $category, int $price) {
        $this->name = $name;
        $this->category = $category;
        $this->price = $price;
    }

    public function getName() : string {
        return $this->name;
    }

    public function getCategory() : string {
        return $this->category;
    }

    public function getPrice() : int {
        return $this->price;
    }
}
?>

File: Second Year/Second Semester/Web Programming/Assignment9-ASP.NET/Models/Product.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace Assignment9_ASP.NET.Models
{
    public class Product
    {
        private long id;
        private string name;
        private string category;
        private int price;
        
        public Product(long id, string name, string category, int price)
        {
            this.id = id;
            this.name = name;
            this.category = category;
            this.price = price;
        }

        public long Id
        {
            get { return id; }
            set { id = value; }
        }

        public string Name
        {
            get { return name; }
            set { name = value; }
        }

        public string Category
        {
            get { return category; }
            set { category = value; }
        }

        public int Price
        {
            get { return price; }
            set { price = value; }
        }
    }
}